manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
> So we have two checks running on database VMs both submitting cf.bbs.Health and cf.bbs.ResonseTime

What happens if one can connect and the other one can't?

> To make check fail, you need to stop both bbs instances

I'm getting `no data` when stopping just one of them.